symfony/client/src/orangehrmAdminPlugin/pages/payGrade/PayGrade.vue

```diff
@@ -0,0 +1,207 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-paper-container">
+      <div class="orangehrm-header-container">
+        <oxd-text tag="h6" class="orangehrm-main-title">
+          Pay Grades
+        </oxd-text>
+        <div>
+        <oxd-button
+          label="Add"
+          iconName="plus"
+          displayType="secondary"
+          @click="onClickAdd"
+          />
+        </div>
+      </div>
+      <table-header
+        :selected="checkedItems.length"
+        :total="total"
+        :loading="isLoading"
+        @delete="onClickDeleteSelected"
+      ></table-header>
+      <div class="orangehrm-container">
```

Comment:
You have to do the following changes to get the desired look in given in mockup. replace `orangehrm-container` with following

```
<div class="orangehrm-background-container">
   <div class="orangehrm-paper-container">
      <div class="orangehrm-header-container">
         <!-- header here-->
      </div>
      <div class="orangehrm-container">
         <!-- table here -->
      </div>
      <!--remove v-if from bottom-container -->
      <div class="orangehrm-bottom-container">
         <!--add v-if to paginator -->
         <oxd-pagination
            v-if="showPaginator"
            :length="pages"
            v-model:current="currentPage"
            />
      </div>
      ...
   </div>
</div>
```

Reply:
Done